packages/web/docs/src/pages/docs/integrations/ci-cd.mdx

```diff
@@ -83,3 +83,39 @@ jobs:
             --registry.accessToken ${{ env.HIVE_TOKEN }} \
             --github
 ```
+
+## Multi Environment Best Practices
+
+By default each project has three targets: `development`, `staging`, and `production`. You can
+utilize these environments (or even add more), to model a multi environment/stage depployment
+process.
+
+For CI, run `hive schema:check`, and for CD, run `hive schema:publish` on the respective target for
```

Comment:
I think the `CI` and `CD` terms are a bit abstract here?

Reply:
I mean, saying "for CI" is a bit vague. I would use something like "For changes in a Pull/Merge Request (CI pipeline), ..."